volumes/miovision/sql/create-table-volumes_daily.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE miovision_api.volumes_daily (
+    intersection_uid integer NOT NULL,
+    dt date NOT NULL,
+    period_start timestamp without time zone NOT NULL,
+    period_end timestamp without time zone NOT NULL,
+    volume_1 integer, 
+    volume_2 integer, 
+    volume_3 integer, 
+    volume_4 integer, 
+    volume_5 integer, 
+    volume_6 integer, 
+    volume_7 integer, 
+    volume_8 integer, 
+    volume_9 integer, 
+    volume_10 integer, 
+    volume_total integer,
+    CONSTRAINT volumes_daily_pkey
+    PRIMARY KEY (intersection_uid, dt)
+);
+
+CREATE INDEX volumes_intersection_idx
+ON miovision_api.volumes_daily
```

Comment:
should we test out hash on this table ? Run a test with btree and hash and see if hash is faster 🤔

Reply:
I  just tested a hash index and there was no difference in performance on this small table. Perhaps we should experiment on a larger one.